src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <_NSwagCommand>$(NSwagExe)</_NSwagCommand>
     <_NSwagCommand
-        Condition="'$(MSBuildRuntimeType)' == 'Core'">dotnet --roll-forward-on-no-candidate-fx 2 $(NSwagDir_Core21)/dotnet-nswag.dll</_NSwagCommand>
+        Condition="'$(MSBuildRuntimeType)' == 'Core'">dotnet --roll-forward-on-no-candidate-fx 2 $(NSwagDir_Core31)/dotnet-nswag.dll</_NSwagCommand>
```

Comment:
I think we need to keep this on v3.0? /cc @dougbu

Reply:
Actually, this should stay at 2.1 to allow the tool to run in as many supported environments as possible.

Reply:
Ok thanks for the reply, will change it back to 2.1